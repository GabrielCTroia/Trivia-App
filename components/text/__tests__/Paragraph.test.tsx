import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import { Paragraph } from '../Paragraph';

it('renders correctly', () => {
  const tree = renderer.create(<Paragraph>Snapshot test!</Paragraph>).toJSON();

  expect(tree).toMatchSnapshot();
});
