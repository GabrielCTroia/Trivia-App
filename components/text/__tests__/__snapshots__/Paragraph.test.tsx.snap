// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Array [
        Object {},
        Array [
          Object {
            "fontSize": 20,
          },
          undefined,
        ],
      ]
    }
  >
    Snapshot test!
  </Text>
</View>
`;
